packages/mds-compliance-service/service/provider.ts

```diff
@@ -0,0 +1,85 @@
+import logger from '@mds-core/mds-logger'
+import { ServiceResult, ServiceException, ServiceProvider, ProcessController } from '@mds-core/mds-service-helpers'
+import { UUID } from '@mds-core/mds-types'
+import { ComplianceSnapshotService, GetComplianceSnapshotsByTimeIntervalOptions } from '../@types'
+import { ComplianceSnapshotRepository } from '../repository'
+import { ValidateComplianceSnapshotDomainModel } from './validators'
+
+export const ComplianceSnapshotServiceProvider: ServiceProvider<ComplianceSnapshotService> & ProcessController = {
+  start: ComplianceSnapshotRepository.initialize,
+  stop: ComplianceSnapshotRepository.shutdown,
+  createComplianceSnapshot: async ComplianceSnapshot => {
```

Comment:
Would not capitalize `ComplianceSnapshot` here, typically we use lowercase first for args `complianceSnapshot`